apps/src/code-studio/levels/dialogHelper.js

```diff
@@ -117,7 +118,7 @@ export function processResults(onComplete, beforeHook) {
         // In this case, errorDialog should be an instance of a React class.
         showDialog(errorDialog);
       } else {
-        showDialog('error', null, adjustScroll);
+        showDialog(<ErrorDialog/>, null, adjustScroll);
```

Comment:
With this change, MatchErrorDialog no longer gets the benefit of adjustScroll, whereas it used to. I think this is probably fine (and we possibly could/should get rid of adjustScroll altogether).

`adjustScroll` was added way back here https://github.com/code-dot-org/code-dot-org/pull/667/files as part of a PR that added the contract_match type. Interestingly, contract_match's errors were later changed to return specific errors, so they wouldn't have actually fallen back to using the generic error + adjustScroll.